modules/application.js

```diff
@@ -0,0 +1,125 @@
+      }</p>`;
+    }, 1000, this);
+  }
+
+      intitializeDocument = () => {
+        this.bookList.innerHTML = '';
+        let i = (this.booksArray.length - 1);
+        this.booksArray.forEach(() => {
+          let bgConst = '';
+          if (i % 2 === 0) {
+            bgConst = 'bg-color';
+          }
+          this.bookList.innerHTML = `${this.bookList.innerHTML}<div class="books-list ${bgConst}">
+          <p>${this.booksArray[i].title} by ${this.booksArray[i].author}</p>
+          <button class="rem_button remove_button_${i}">remove</button>
+          </div>`;
+          i -= 1;
+        }, i);
+      }
+
+      removeBook = (index) => {
+        document.querySelector(`.remove_button_${index}`).addEventListener('click', (event) => {
+          const tempbooksArray = [];
+          const objectReference = event.currentTarget.ref;
+          this.booksArray.forEach((element, j) => {
+            if (j !== event.currentTarget.index) {
+              tempbooksArray.push({
+                title: objectReference.booksArray[j].title,
+                author: objectReference.booksArray[j].author,
+              });
+            }
+          }, tempbooksArray, objectReference);
+          objectReference.booksArray = tempbooksArray;
+          localStorage.setItem('books', JSON.stringify(objectReference.booksArray));
+          objectReference.intitializeDocument();
+          objectReference.intitializeRemoveButtonEvents();
+        });
+        document.querySelector(`.remove_button_${index}`).index = index;
+        document.querySelector(`.remove_button_${index}`).ref = this;
+      }
+
+      intitializeRemoveButtonEvents = () => {
+        let i = (this.booksArray.length - 1);
+        this.booksArray.forEach(() => {
+          this.removeBook(i);
+          i -= 1;
+        }, i);
+      }
+
+      addBook = () => {
+        this.displayBookListSection();
+        this.booksArray.push({ title: this.title.value, author: this.author.value });
+        localStorage.setItem('books', JSON.stringify(this.booksArray));
+        this.title.value = '';
+        this.author.value = '';
+        this.intitializeDocument();
+        this.intitializeRemoveButtonEvents();
+      }
+
+      displayBookListSection = () => {
+        this.bookListSection.classList.remove('vanish');
+        this.addBookSection.classList.add('vanish');
+        this.contactSection.classList.add('vanish');
+      }
+
+      displayAddBookSection = () => {
+        this.bookListSection.classList.add('vanish');
+        this.addBookSection.classList.remove('vanish');
+        this.contactSection.classList.add('vanish');
+      }
+
+      displayContactSection = () => {
+        this.bookListSection.classList.add('vanish');
+        this.addBookSection.classList.add('vanish');
+        this.contactSection.classList.remove('vanish');
+      }
+}
+/* eslint-disable */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.